bin/check_govet

```diff
@@ -22,4 +22,4 @@ fi
 go vet -composites=false ${ARGUMENTS}
 
 print_success "All files are compliant to go vet."
-exit 0
\ No newline at end of file
+exit 0
```

Comment:
I guess exit 0 can be removed here. As it's returned by default
Also add ```set -e flag``` at the beginning of the script - script will fail on any failing subcommand

Reply:
> As it's returned by default

I disagree on that one @tadovas, Bash scripts exit with the status of the last command executed I believe.  You are right though if we use `set -e` then `exit 0` is redundant.  Sounds like we want project wide `set -e` in all project Bash scripts, no use of `exit 0`, and a newline terminating each file?

Reply:
That default part I meant result of last command, thanks for correction. Well using set -e project wide is maybe too much, it depends on situation, but in this particular case I don't see a point to use explicit exit 0.